superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterEditPopoverSimpleTabContent/index.jsx

```diff
@@ -41,6 +41,10 @@ import AdhocFilter, {
 import columnType from 'src/explore/components/controls/FilterControl/columnType';
 
 const SelectWithLabel = styled(Select)`
+  .ant-select-selector {
```

Comment:
This seems fine, but I'm curious about one potential pain point... 
Is this likely to crop up in other Popover contexts? I'm not sure how the other instances DON'T have this problem. 
Wondering if this margin should be applied (with a prop?) the the Select itself, or if there's some other sort of "global" fix we're missing out on here.

Or am I over thinking it, and this will only be needed in this context?

Reply:
This is a fine bandaid for now... just wondering if this is something worth revisiting later for less code and/or greater consistency.

Reply:
This crossed my mind as well, but I couldn't find any other examples with the problem. Lacking multiple examples, I opted not to generalize the solution.